lib/tests/streamlit/elements/help_test.py

```diff
@@ -193,23 +190,171 @@ def test_doc_defined_for_type(self):
+            "test_passing_a_class.<locals>.MyClass()",
+            ds.value,
+        )
+        self.assertEqual("class", ds.type)
+        self.assertEqual("", ds.doc_string)
+
+    def test_padding_an_instance(self):
         """When the type of the object is type and no docs are defined,
-        we expect docs are not available"""
+        we expect docs to be None."""
 
         class MyClass(object):
             pass
 
-        st.help(MyClass)
+        with patch_varname_getter():
+            st.help(MyClass)
 
         ds = self.get_delta_from_queue().new_element.doc_string
         self.assertEqual(type(MyClass), type)
         self.assertEqual("MyClass", ds.name)
-        self.assertEqual("tests.streamlit.elements.help_test", ds.module)
-        self.assertEqual("No docs available.", ds.doc_string)
+        self.assertEqual(
+            "tests.streamlit.elements.help_test.StHelpTest."
+            "test_padding_an_instance.<locals>.MyClass()",
+            ds.value,
+        )
+        self.assertEqual("class", ds.type)
+        self.assertEqual("", ds.doc_string)
+
+    def test_class_members(self):
+        class MyClass(object):
+            a = 1
+            b = 2
+
+            def __init__(self):
+                self.c = 3
+                self.d = 4
+
+            @property
+            def e(self):
+                "Property e"
+                return 5
+
+            @staticmethod
+            def staticmethod1(self, x=10):
+                "Static method 1"
+
+            @classmethod
+            def classmethod1(self, y=20):
+                "Class method 1"
+
+        with patch_varname_getter():
+            st.help(MyClass)
+
+        ds = self.get_delta_from_queue().new_element.doc_string
+        self.assertEqual(len(ds.members), 5)
+
+        self.assertEqual(ds.members[0].name, "a")
+        self.assertEqual(ds.members[0].value, "1")
+        self.assertEqual(ds.members[0].doc_string, "")
+        self.assertEqual(ds.members[0].type, "int")
+
+        self.assertEqual(ds.members[1].name, "b")
+        self.assertEqual(ds.members[1].value, "2")
+        self.assertEqual(ds.members[1].doc_string, "")
+        self.assertEqual(ds.members[1].type, "int")
+
+        self.assertEqual(ds.members[2].name, "e")
+        self.assertEqual(ds.members[2].value, "")
+        self.assertEqual(ds.members[2].doc_string, "Property e")
+        self.assertEqual(ds.members[2].type, "property")
+
+        self.assertEqual(ds.members[3].name, "classmethod1")
+        self.assertEqual(ds.members[3].value, "")
+        self.assertEqual(ds.members[3].doc_string, "Class method 1")
+        self.assertEqual(ds.members[3].type, "method")
+
+        self.assertEqual(ds.members[4].name, "staticmethod1")
+        self.assertEqual(ds.members[4].value, "")
+        self.assertEqual(ds.members[4].doc_string, "Static method 1")
+        self.assertEqual(ds.members[4].type, "function")
```

Comment:
nit: this could be made more concise by writing something like

```python
expected_members = [
    { 'name': 'a', 'value': '1', 'doc_string': '', 'type': 'int' },
    # ...
]

for i in range(len(expected_members)):
    self.assertEqual(ds.members[i].name, expected_members['name'])
    # ...
```